packages/lodestar/test/unit/api/rest/index.test.ts

```diff
@@ -0,0 +1,54 @@
+import {SinonStubbedInstance} from "sinon";
+import {config} from "@chainsafe/lodestar-config/mainnet";
+
+import {ApiNamespace, RestApi} from "../../../../src/api";
+import {StubbedApi} from "../../../utils/stub/api";
+import {testLogger} from "../../../utils/logger";
+import {DebugBeaconApi} from "../../../../src/api/impl/debug/beacon";
+import {BeaconStateApi} from "../../../../src/api/impl/beacon/state";
+
+export const BEACON_PREFIX = "/eth/v1/beacon";
+export const CONFIG_PREFIX = "/eth/v1/config";
+export const NODE_PREFIX = "/eth/v1/node";
+export const VALIDATOR_PREFIX = "/eth/v1/validator";
+
+beforeEach(async function () {
```

Comment:
As it stands right now, it just attaches the objects to `this` in these outer tests which are then accessed inside of each individual test via `this.ctx` or `this.test.ctx`, depending on where you're accessing it.  But also, yeah I just realized that if you declare more than one of these index.test.ts with the beforeEach and afterEach in the global scope, then it calls both even if they are in different directories.  So, maybe it would be better to find a way to make sure that each index.test.ts is only ran for the tests in their corresponding directories (i.e. `rest/` and `impl/`)